File: api-server/models/Album.js
const mongoose = require('mongoose')
const Artist = require('../models/Artist')

const albumSchema = mongoose.Schema({
    _id: String,
    album_type : String,
    artists : {
        type: String,
        ref: 'Artist'
    },
    available_markets : [String],
    external_urls: { 
        spotify: String
    },
    href : String,
    images: [{
        height : String,
        url : String,
        width : String
    }],
    name : String,
    release_date : String,
    release_date_precision : String,
    total_tracks : Number,
    type : String,
    uri : String
}, { versionKey: false })

module.exports = mongoose.model('Album', albumSchema)

File: app/src/main/java/com/lucianghimpu/matchmefy/data/dataModels/Album.kt
package com.lucianghimpu.matchmefy.data.dataModels

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Album (
    val artists : List<Artist>,
    val external_urls : ExternalUrls,
    val id : String,
    val images : List<Image>,
    val name : String,
    val uri : String
) : Parcelable